megazords/full/src/lib.rs

```diff
@@ -13,7 +13,6 @@ pub use crashtest;
 pub use fxa_client;
 pub use logins;
 pub use nimbus;
-pub use places_ffi;
```

Comment:
should this be replaced with `places`? (dito in the `Cargo.toml`) I'm a little concerned that our CI didn't catch this 😅 unless it's somehow imported some other way? or do our android tests not run against the real megazord?

Reply:
lol - nice spot - it turns out places still gets sucked in via sync-manager - so we could kill those other uses too! However, I think it's better to be clear. (FWIW, I did strike this exact same problem for iOS, but it doesn't use the sync manager, so the fix there was to do what you suggest!)